components/Layout/HomeHeader/HomeHeaderItems.vue

```diff
@@ -204,9 +200,20 @@ export default Vue.extend({
   },
   mounted() {
     this.swiper = (this.$refs.homeSwiper as Vue).$swiper as Swiper;
+    const hash = this.getBlurhashHash(this.items[0], ImageType.Backdrop);
```

Comment:
```suggestion
    const hash = this.getBlurhash(this.items[0], ImageType.Backdrop);
```

It feels a bit redundant with two hashes.